src/domain/DomainController.h

```diff
@@ -4,8 +4,8 @@
 /// \author     My Linh Wuerzburger
 /// \copyright  <2015-2020> Forschungszentrum Juelich GmbH. All rights reserved.
 
-#ifndef ARTSS_BOUNDARY_BOUNDARYCONTROLLER_H_
-#define ARTSS_BOUNDARY_BOUNDARYCONTROLLER_H_
+#ifndef ARTSS_DOMAIN_BOUNDARYCONTROLLER_H_
```

Comment:
why is it called `ARTSS_DOMAIN_BOUNDARYCONTROLLER_H_` in `DomainController.h`?

Reply:
33437f9